UI/Pages/Enquiry/Build/CheckYourAnswers.cshtml.cs

```diff
@@ -95,8 +95,8 @@ public async Task<IActionResult> OnPostAsync()
 
         if (!submittedConfirmationModel.IsValid && submittedConfirmationModel.ErrorStatus == HttpStatusCode.InternalServerError.ToString())
         {
-            TempData["Status"] = HttpStatusCode.InternalServerError;
-            return RedirectToPage(nameof(ErrorModel));
+            //This is a temporary fix, until errors are addressed in NTP-1044
+            throw new InvalidDataException("Error on check you answers");
```

Comment:
We can create an `InternalServerErrorResult` class and redirect the user to the Error page as we do for the 404 `NotFound` stuff.. i.e: 
```
/// <summary>
/// Represents an <see cref="StatusCodeResult"/> that when
/// executed will produce a Not Found (500) response.
/// </summary>
/// 
[DefaultStatusCode(DefaultStatusCode)]
public class InternalServerErrorResult : StatusCodeResult
{
    private const int DefaultStatusCode = StatusCodes.Status500InternalServerError;

    /// <summary>
    /// Creates a new <see cref="NotFoundResult"/> instance.
    /// </summary>
    public InternalServerErrorResult() : base(DefaultStatusCode)
    {
    }
}
```
We can just call `return new InternalServerErrorResult(),` which will redirect the user to the Error page with a 500 status code.

Reply:
This is a temporary fix, until errors are addressed in NTP-1044.  With my intention being that unexpected exceptions will be allowed to propagate.